### DISCUSSION TITLE
Does web-based editor at github.dev support any startup configuration?

### DISCUSSION THREAD

**Discussion by kenorb** [community/community#19975] [Codespaces] [answered]
I would like to automate installation of few extensions for web-based editor at github.dev on startup, something similar to `devcontainer.json`.

I'm not really after _Settings sync_, it's more for extensions needed for particular project, so anybody opening the project in web-based editor, will have these extensions installed.

Is it possible?

**Comment by diegoleondev** [comment, marked as answer]
Hi @kenorb, for security it is not possible to install default extensions, what you can do is indicate in the `package.json` the dependencies:
```json
"extensionDependencies": [
    "extension.id",
    "esbenp.prettier-vscode"
  ]
```

or 

```
"extensionPack": [
    "extensionPack.id",
    "vscjava.vscode-java-pack"
]
```

In case of not using `package.json` the best thing you can do is include details in the `README.md`

> **Warning**
> Extensions required
> - Prettier
> - EsLint
